### PR TITLE
Shownig, adding and changing quantity of the book

### PR DESCRIPTION
**Pull request by jnapo**

### CODE REVIEW

File: src/components/Book.tsx [hunk 1]
```diff
@@ -23,7 +26,23 @@ const Book: FC<IBookProps> = ({
   description,
   isbn,
 }) => {
-  const [quantity, setQuantity] = useState('1');
+  const [data, setData] = useState<IBook[] | null>([]);
+  const [quantityInput, setQuantityInput] = useState('1');
+
+  useEffect(() => {
+    const getAllBooksId = async () => {
```
Comment (DLowHP) [inline src/components/Book.tsx:33]: I think it should be named "getBookData" or something similar

File: src/components/UpdateBook.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { supabase } from '../utils/supabase';
+import { IBook } from '../interfaces/IBook.interface';
+
+interface IBookProps {
+  id?: string;
+  quantity?: number;
+}
+
+const AddBook: FC<IBookProps> = ({ id, quantity }) => {
```
Comment (DLowHP) [inline src/components/UpdateBook.tsx:10]: Did you mean "UpdateBook"? 
```suggestion
const UpdateBook: FC<IBookProps> = ({ id, quantity }) => {
```

File: src/components/UpdateBook.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        transition duration-500 ease-in-out hover:bg-gray-400 hover:text-gray-100"
+        onClick={Update}
+      >
+        + Zwiększ ilość w magazynie
+      </button>
+    </div>
+  );
+};
+
+export default AddBook;
```
Comment (DLowHP) [inline src/components/UpdateBook.tsx:32]: Same thing here
```suggestion
export default UpdateBook;
```

Comment (jnapo) [reply src/components/UpdateBook.tsx]: Done

**Pull request merged**